.cicd/platforms/macos-10.14.sh

```diff
@@ -49,6 +49,17 @@ if [[ ! $PINNED == false || $UNPINNED == true ]]; then
     sudo make install
     cd ../..
     rm -rf clang8
+    # install boost from source
+    curl -LO https://dl.bintray.com/boostorg/release/1.70.0/source/boost_1_70_0.tar.bz2
+    tar -xjf boost_1_70_0.tar.bz2
+    cd boost_1_70_0
+    ./bootstrap.sh --with-toolset=clang --prefix=/usr/local
+    sudo ./b2 toolset=clang cxxflags='-stdlib=libc++ -D__STRICT_ANSI__ -nostdinc++ -I/usr/local/include/c++/v1' linkflags='-stdlib=libc++' link=static threading=multi --with-iostreams --with-date_time --with-filesystem --with-system --with-program_options --with-chrono --with-test -q -j$(getconf _NPROCESSORS_ONLN) install
```

Comment:
This is different than what the old script did; looks like you copied the parameters that are used when building pinned Linux boost. Not sure if that's okay or not.

Reply:
I'll review logs from the old script and update shortly. Thanks for noticing!

Reply:
Old logs confirm comments from @spoonincode above. Boost parameters have been adjusted to match those used in the base image pipeline.